lib/canary/plugs.ex

```diff
@@ -193,11 +193,12 @@ defmodule Canary.Plugs do
     current_user = Dict.fetch! conn.assigns, current_user_name
     action = get_action(conn)
     is_persisted = persisted?(opts)
+    non_id_actions = if opts[:non_id], do: ([:index, :new, :create] ++ opts[:non_id]), else: [:index, :new, :create]
```

Comment:
Also, I feel like `opts[:non_id_actions]` would be more clear than `opts[:non_id]`.